ampd/src/handlers/config.rs

```diff
@@ -142,6 +147,22 @@ where
     }
 }
 
+fn validate_starknet_msg_verifier_config<'de, D>(configs: &[Config]) -> Result<(), D::Error>
+where
+    D: Deserializer<'de>,
+{
+    match configs
```

Comment:
optional: looks to me like a simple `if` would be easier to understand.

Reply:
I agree, but all functions are written like that, so it's a copy paste effort (like some other things).
I prefer to not make our one the only different one.